packages/paypal-js/types/components/card-fields.d.ts

```diff
@@ -1,3 +1,5 @@
+import { OnApproveData } from "./buttons";
```

Comment:
I worry about sharing the exact same onApproveData object from buttons. Could we have cardFields have its own unique type for it? That way they can be independent in the future and any future changes need to be explicit for both.

Reply:
👍 they may be slightly different. if we end up with 3+ onApprove definitions we could at that time write some type of `baseOnApproveData`

Reply:
Ahh that's a great idea. 😄